src/hooks/useCoinListEditOptions.js

```diff
@@ -52,6 +53,9 @@ export default function useCoinListEditOptions() {
     [dispatch]
   );
 
+  // Clear selection queue on unmount.
+  useEffect(() => () => dispatch(clearSelectedCoins()), [dispatch]);
```

Comment:
Don't you have to pass an empty array to run onmount / unmount?

Reply:
I would guess that dispatch's function identity stays stable such that you technically don't even need it in the useEffect (but bc it's coming from react-redux useDispatch instead of react's useReducer, eslint will show a warning). So leaving it in the dependency array will still behave as we expect + not have eslint complain (this is all assuming react-redux made their dispatch stable)

Reply:
yes! @jinchung is correct!